Assets/Tests/PlayMode/EOSConfigTests.cs

```diff
@@ -218,7 +218,7 @@ public void DeploymentOverride_MustBeValid()
                 }
             };
 
-            if (!config.TryGetFailingValidatorAttributes(out List<FieldValidatorFailure> failingAttributes))
+            if (!FieldValidator.TryGetFailingValidatorAttributes(config.sandboxDeploymentOverrides[0], out List<FieldValidatorFailure> failingAttributes))
```

Comment:
Here, I think we should run this on both the config and on the targeted override. Running it on the config ensures that the config is checking its innards for more problems.

Reply:
This will be updated alongside the recursive validate task